core/modules/system/css/system.css

```diff
@@ -60,12 +60,6 @@
  *
  * @see collapse.js
  */
-.js fieldset.collapsed {
```

Comment:
Checked cleanish, radix, bartik, and aeon5 and I'm not seeing any negative side effects in their themes from removing this.